<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1955</width>
    <height>1013</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1 Exp: 1500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2 Exp:1500</string>
    </property>
   </widget>
   <widget class="QLabel" name="dice_1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dice_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Dice</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>409</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/goku.jpeg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="player_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Player2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>1162</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>43</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Players           How Many Human?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Move</string>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="avail">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Avail?</string>
    </property>
   </widget>
   <widget class="QLabel" name="rentPrice">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Rent Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchase</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlayer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Player = None</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Player2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="movieLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="properties1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>680</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1 Properties: </string>
    </property>
   </widget>
   <widget class="QLabel" name="properties2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>710</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2 Properties: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>270</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalExp">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>150</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exp Distributed: 3000</string>
    </property>
   </widget>
   <widget class="QLabel" name="challenge1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Challenge: 0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="challenge2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Adventure: 0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="totalprop1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>220</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 number of Properties: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalprop2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>240</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P2 number of Properties: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1955</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
